.github/workflows/pr-bot-assistant.yml

```diff
@@ -0,0 +1,96 @@
+name: Bot Assistant
```

Comment:
I prefer a specific workflow: `Update windows fixtures` so we don't have to mentally check which action will `@babel-bot` accept.

If multiple workflows share the same step, we can always abstract them into a workflow generator, but don't have to manually provide `update-windows-fixtures`.